modules/widgets/package.json

```diff
@@ -25,6 +25,10 @@
       "types": "./dist/index.d.ts",
       "import": "./dist/index.js",
       "require": "./dist/index.cjs"
+    },
+    "./stylesheet.css": {
+      "import": "./dist/stylesheet.css",
+      "style": "./dist/stylesheet.css"
```

Comment:
I can't find any documentation about this field. I do see some libraries set the top-level `style` field, though, like [bootstrap](https://unpkg.com/browse/bootstrap@5.3.3/package.json). Maybe that is what we need?

Reply:
Perhaps [`default`](https://nodejs.org/api/packages.html#conditional-exports) could replace both `import` and `style` here? I believe this change is currently fine for ESM users but may break for CJS users. The subset of bundler configs that are still using CJS but are new enough to import stylesheets might be small.

I'm having trouble finding information on `package.json#style` but I think the idea is that resolves things like imports within PostCSS modules. A bundler _may_ inline your dependencies' `style` fields by default, but may not:

https://github.com/parcel-bundler/parcel/issues/432

Reply:
@atmorling could you point us to where found this solution? Maybe this style tag should be removed if it doesn't seem to do anything.

I can go either way on the top-level `style` since it doesn't seem standardized, though I'm leaning towards doing what can be tested (add a post-css test app?)

Reply:
I found it in a webpack [guide](https://webpack.js.org/guides/package-exports/#guidelines) on package.json documentation, but you're right, it doesn't appear in Node's documentation at all. It should probably be removed.